sfs/mono/__init__.py

```diff
@@ -3,14 +3,23 @@
 .. autosummary::
     :toctree:
 
-    drivingfunction
+    esaedge
+    nfchoa
+    sdm
+    wfs
+
     source
```

Comment:
What about placing `source` before the driving functions?

Reply:
yes, that seems to be a good categorizing style.